org.getmonero.Monero.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: org.getmonero.monero.desktop
+finish-args:
+  - --socket=x11
+  - --share=network
+  - --device=dri
+  - --share=ipc
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: monero-wallet-gui
+modules:
+  - name: monero-bin
+    buildsystem: simple
+    sources:
+    - type: archive
+      url: https://github.com/monero-project/monero-gui/archive/refs/tags/v0.17.1.9.tar.gz
+      sha256: 3a924b4e08b557d93337a0c05826a9fa95e5cf8c503aff0ecb3676be45db1d14
+    build-commands:
+    - tar xzf v0.17.1.9.tar.gz
+    - cd monero-gui-0.17.1.9
+    - docker build --tag monero:build-env-linux --build-arg THREADS=4 --file Dockerfile.linux .
```

Comment:
I doubt this will work. Also there is no network access in the build sandbox

Reply:
I added the `docker build` line and `docker run` line because that is what the official Monero Desktop GUI build instructions say right now.

```
### Building Reproducible Linux static binaries with Docker (any OS)

1. Install Docker [https://docs.docker.com/engine/install/](https://docs.docker.com/engine/install/)
2. Clone the repository
   ```
   git clone --branch master --recursive https://github.com/monero-project/monero-gui.git
   ```
   \* `master` - replace with the desired version tag (e.g. `v0.17.1.9`) to build the release binaries.
3. Prepare build environment
   ```
   cd monero-gui
   docker build --tag monero:build-env-linux --build-arg THREADS=4 --file Dockerfile.linux .
   ```
   \* `4` - number of CPU threads to use

4. Build
   ```
   docker run --rm -it -v <MONERO_GUI_DIR_FULL_PATH>:/monero-gui -w /monero-gui monero:build-env-linux sh -c 'make release-static -j4'
   ```
   \* `<MONERO_GUI_DIR_FULL_PATH>` - absolute path to `monero-gui` directory  
   \* `4` - number of CPU threads to use
5. Monero GUI Linux static binaries will be placed in  `monero-gui/build/release/bin` directory
```

Ref: https://raw.githubusercontent.com/monero-project/monero-gui/master/README.md

It seems like it is better to use their pre-compiled builds for Linux x86_64 if there is not proper network access in the flathub build sandbox.

Reply:
Nope. it's better to do without Docker.